File: eternal_return/src/main/java/org/EternalReturn/ERCharacter/Character/NullCharacter.java
package org.EternalReturn.ERCharacter.Character;

import org.EternalReturn.ERCharacter.ERCharacter;
import org.EternalReturn.ERPlayer.ERPlayer;

public class NullCharacter extends ERCharacter {
    public NullCharacter(ERPlayer erPlayer) {
        super(erPlayer);
    }

    @Override
    public String getName() {
        return "NullCharacter.";
    }
}

File: eternal_return/src/main/kotlin/org/EternalReturn/ERCharacter/character/NullCharacter.kt
package org.EternalReturn.ERCharacter.character

import org.EternalReturn.ERCharacter.ERCharacter
import org.EternalReturn.ERPlayer.ERPlayer

class NullCharacter(erPlayer: ERPlayer) : ERCharacter(erPlayer) {
    override val name: String
        get() = "NullCharacter."
}

